tests/benchmark/bm_files.sh

```diff
@@ -26,6 +26,12 @@ elif [ "$BM_TYPE" = "bm-basics-fp16-single" ] \
 || [ "$BM_TYPE" = "bm-batch-iter-fp16-multi" ]
 then
     file_name="basic_fp16"
+elif [ "$BM_TYPE" = "bm-basics-int8-single" ] \
+|| [ "$BM_TYPE" = "bm-basics-int8-multi" ] \
```

Comment:
Benchmarks for multi will come in a different PR no?

Reply:
Yes but it will be needed, so I added it now.

Reply:
Just for the good order, I would suggest adding it with the PR  that will introduce multi and not now.